src/widget/wslidercomposed.cpp

```diff
@@ -190,8 +190,14 @@ double WSliderComposed::calculateHandleLength() {
                     mode == Paintable::TILE || m_pHandle->height() == 0.0) {
                 return m_pHandle->width();
             } else if (mode == Paintable::STRETCH_ASPECT) {
+                const qreal handleHeight =
+                    static_cast<double>(m_pHandle->height());
+                if (fabs(handleHeight) < 1.0e-10) {
```

Comment:
Hm, yeah, maybe. Using `fabs` seemed somewhat safer to me, but I have no strong opinion about this. Your call.